ccpp/suites/suite_FV3_GFS_v15_nssl_mynn_lam3km.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_nssl_mynn_lam3km" version="1">
```

Comment:
Why do we add this suite file? Is there any test using this suite file?

Reply:
It's one that we have been using, but I suppose it doesn't need to be in this repository. I can take it out.

Reply:
If you're using it, I think it is important to keep it in -- probably more important than the one used in RTs.

Reply:
Yes, it's good to keep it if you are using it. Thanks